File: main/src/main/java/net/fwitz/math/main/binary/split/functions/exponential/SplitExpZ.java
package net.fwitz.math.main.binary.split.functions.exponential;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitExpZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("exp(z)")
                .computeFn(SplitComplex::exp)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/exponential/SplitExpZ.kt
package net.fwitz.math.main.binary.split.functions.exponential

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitExpZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("exp(z)")
        .computeFn(SplitComplex::exp)
        .render()
}